modules/command-engine/core/src/main/scala/surge/core/commondsl/SurgeGenericBusinessLogicTrait.scala

```diff
@@ -31,7 +32,9 @@ trait SurgeGenericBusinessLogicTrait[AggId, Agg, Command, Rej, Event] {
 
   def metrics: Metrics = Metrics.globalMetricRegistry
 
-  def tracer: Tracer = NoopTracerFactory.create()
+  val openTelemetry: OpenTelemetry = OpenTelemetry.noop()
+
+  private[surge] def tracer: Tracer = openTelemetry.getTracer(OpenTelemetryInstrumentation.Name, OpenTelemetryInstrumentation.Version)
```

Comment:
This is an important change API change.
Users no longer provide a "tracer" but an OpenTelemetry instance. We get the tracer from the OpenTelemetry instance and we name it (multiple libs can get tracers from the same OpenTelemetry instance). 
See here: https://opentelemetry.io/docs/java/manual_instrumentation/  ("if you are writing library instrumentation, it is strongly recommended that you provide your users the ability to inject an instance of OpenTelemetry into your instrumentation code.")